lib/util.js

```diff
@@ -151,7 +152,24 @@ module.exports.readFilesToArray = function(fileList) {
   var contentArray = [];
   fileList.forEach(function(filePath) {
     try {
-      contentArray.push(fs.readFileSync(filePath));
+      let extension = filePath.split('.').pop();
+      let content = fs.readFileSync(filePath);
+      if(extension == 'p12'){
```

Comment:
`pfx` extension is a synonym for `p12`. Should the condition take into account both extensions?

Reply:
Good catch!